templates/solutions/open-source-security/cyber-resilience-act/form-data.json

```diff
@@ -0,0 +1,59 @@
+{
+  "form": {
+    "/solutions/open-source-security/cyber-resilience-act": {
+      "templatePath": "/solutions/open-source-security/cyber-resilience-act/index.html",
+      "isModal": true,
+      "modalId": "cra-contact-modal",
+      "formData": {
+        "title": "Get in touch with our experts on the Cyber Resilience Act (CRA) compliance",
+        "formId": "6051",
+        "returnUrl": "https://canonical.com/solutions/open-source-security/cyber-resilience-act#contact-form-success",
+        "product": ""
+      },
+      "fieldsets": [
+        {
+          "title": "Fill in this form and a member of our team will be in touch shortly.",
+          "id": "contact-form",
```

Comment:
is it missing "noCommentsFromLead" field here? cause otherwise this title is being added to the comments section of the Marketo API payload.
![Screenshot From 2025-01-14 15-50-58](https://github.com/user-attachments/assets/0633ca6a-4c5a-4dcc-8e7c-bf62674a34c6)

Reply:
You are right, good catch!